### PR TITLE
terranix: init at 2.3.0

### PR DESCRIPTION
**Pull request by mrVanDalo**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mrVanDalo** [comment]
second try to add terranix to nixpkgs (#68019)

**Comment by Lassulus** [comment]
this probably needs to be called from all-packages.nix

File: pkgs/applications/networking/cluster/terranix/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+stdenv.mkDerivation {
+
+  pname = "terranix";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "mrVanDalo";
+    repo = "terranix";
+    rev = "2.3.0";
```
Comment (nh2) [inline pkgs/applications/networking/cluster/terranix/default.nix:11]: Maybe reuse `version`?

```suggestion
    rev = version;
```

File: pkgs/applications/networking/cluster/terranix/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    sha256 = "030067h3gjc02llaa7rx5iml0ikvw6szadm0nrss2sqzshsfimm4";
+  };
+
+  installPhase = ''
+    mkdir -p $out/{bin,core,modules,lib}
+    mv bin core modules lib $out/
```
Comment (nh2) [inline pkgs/applications/networking/cluster/terranix/default.nix:17]: I think the arguments to `mkdir` do not make sense, as your `mv` immediately afterwards overwrite them. How about this simpler version?

```suggestion
    mkdir -p $out
    mv bin core modules lib $out/
```

**Review by nh2 — APPROVED** [review]

**Pull request merged**